widgets/bat_acpi_freebsd.lua

```diff
@@ -0,0 +1,53 @@
+-----------------------------------------------------
+-- Battery: use acpiconf to get more information
+--          from the battery
+-----------------------------------------------------
+local setmetatable = setmetatable
+
+local bat_acpi = {}
+
+local function worker(format)
+    local bat_info = {}
+    local pcall = "acpiconf -i batt"
```

Comment:
I would suggest to `"acpiconf -i " .. batt` and adding a function argument to choose which battery is addressed. Though most notebooks have only one battery ... what do you think?